cpp/tests/experimental/spatial/linestring_distance_test.cu

```diff
@@ -19,23 +19,32 @@
 #include <cuspatial/error.hpp>
 #include <cuspatial/experimental/iterator_factory.cuh>
 #include <cuspatial/experimental/linestring_distance.cuh>
+#include <cuspatial/experimental/ranges/multilinestring_range.cuh>
 #include <cuspatial/vec_2d.hpp>
 
-#include <rmm/device_uvector.hpp>
 #include <rmm/device_vector.hpp>
 
 #include <thrust/iterator/constant_iterator.h>
 #include <thrust/iterator/counting_iterator.h>
 
+#include <initializer_list>
+
 namespace cuspatial {
 namespace test {
 
-using namespace cudf;
+template <typename T>
+auto make_device_vector(std::initializer_list<T> inl)
```

Comment:
Ooh, this looks handy. Perhaps it could be moved to a test utility header...

Reply:
Done